realm/realm-library/src/objectServer/java/io/realm/mongodb/sync/SyncConfiguration.java

```diff
@@ -1012,6 +1019,28 @@ public Builder maxNumberOfActiveVersions(long number) {
             return this;
         }
 
+        /**
+         * Sets whether or not users are allowed to perform calls to {@link Realm#executeTransaction} from the UI thread.
+         * <p>
```

Comment:
Any changes to the Javadoc in `RealmConfiguration.Builder()` should be reflected here.